waspc/cli/Main.hs

```diff
@@ -84,6 +86,25 @@ printUsage =
 printVersion :: IO ()
 printVersion = putStrLn $ showVersion version
 
+-- dynamically generate available option for wasp bash completion script tools/bash-completion
+bashCompletion :: [String] -> IO ()
+bashCompletion args = case args of
+  ["all"] -> listCommands
```

Comment:
I would try to avoid using `all` as it seems somewhat magical -> it would be somewhat nicer if it was just not args in this case.

Reply:
You mean it would be like this
```haskell
bashCompletion args = case args of
  [""] -> listCommands
```

Reply:
Actually, it would probably be `[]` I think?